deploy/charts/cert-manager/README.md

```diff
@@ -112,6 +112,14 @@ The following table lists the configurable parameters of the cert-manager chart
 | `webhook.caSyncImage.repository` | CA sync image repository | `quay.io/munnerz/apiextensions-ca-helper` |
 | `webhook.caSyncImage.tag` | CA sync image tag | `v0.1.0` |
 | `webhook.caSyncImage.pullPolicy` | CA sync image pull policy | `IfNotPresent` |
+| `cainjector.enabled` | Toggles whether the cainjector component should be installed (required for the webhook component to work) | `true` |
+| `cainjector.replicaCount` | Number of cert-manager cainjector replicas | `1` |
```

Comment:
I'm confused, why do we need this?

Reply:
The README? Or the replica count? Because it runs as a separate/standalone deployment, which has its own replica count field 😄